src/com/amazon/corretto/crypto/provider/Utils.java

```diff
@@ -572,4 +572,27 @@ static <T> T requireNonNull(final T obj, final String message) {
   static String requireNonNullString(final String s, final String message) {
     return requireNonNull(s, message);
   }
+
+  enum NativeContextReleaseStrategy {
+    HYBRID,
+    LAZY,
+    EAGER
+  }
+
+  static NativeContextReleaseStrategy getNativeContextReleaseStrategyProperty(
+      final String propertyName) {
```

Comment:
This interface doesn't make sense. When would you ever need to call this with a different property name? Supporting that seems error-prone. Suggest referencing a constant within this method and removing the indirection inside AmazonCorrettoCryptoProvider.

Reply:
Done.